src/blueapi/core/device_walk.py

```diff
@@ -0,0 +1,17 @@
+from typing import Iterable, Tuple
+
+from blueapi.core import Device, HasComponents
+
+DeviceTreeNode = Tuple[Device]
+
+
+def walk_devices(devices: Iterable[Device]) -> Iterable:
```

Comment:
Where is this used?

Reply:
Nowhere any more, was used in a previous iteration of a prototype.

Reply:
Should be deleted

Reply:
In fact I think this whole file can go